internal/tpl/funcs.go

```diff
@@ -27,6 +30,9 @@ const (
 	FuncGet         = "get"
 	FuncGetPassword = "getpw"
 	FuncGetValue    = "getval"
+	FuncArgon2i     = "argon2i"
```

Comment:
Please document these new functions in the Template.md documentation.

Reply:
Done.